AppCreationScripts/apps.json

```diff
@@ -1,44 +0,0 @@
-{
```

Comment:
This file should absolutely not be removed, as it will break the UWP quickstart in the portal.
Please re-add it.

Reply:
This is our UWP quickstart? :( It hasn't been updated since MSAL 4.25 :(